web/src/message_edit.ts

```diff
@@ -1400,13 +1498,17 @@ export function is_message_oldest_or_newest(
         ]),
     };
 
-    channel.get({
+    void channel.get({
         url: "/json/messages",
         data,
         success(data) {
+            // TODO:  Sync this with the actual data structure we receive here.
```

Comment:
I think actually we only are interested in the message IDs returned here, so I'm not sure we want to parse more strictly.

Reply:
Ah okay, I'd thought maybe we want to make sure they were in sync, but if we don't need to parse the full data structure I can remove this TODO.